workflows/cve-fixer/.claude/commands/cve.find.md

```
@@ -0,0 +1,556 @@
+     - Component: user-specified component (from step 2)
+     - Filter: Issues with "CVE-" pattern in any text field
+   - Example MCP tool usage (adapt based on actual MCP server available):
+     - Look for tools like: `query_jira_issues`, `search_jira`, `jira_jql_search`, etc.
+     - Pass JQL query: `project = RHOAIENG AND component = "COMPONENT_NAME" AND text ~ "CVE-*"` (append `AND status not in ("Resolved")` if `--ignore-resolved` flag was provided)
+     - Request columns/fields: KEY, SUMMARY, STATUS, PRIORITY, CREATED, COMPONENTS
+   - **If 0 results returned**: print a message noting the component may be misspelled or have no CVEs filed, and stop — do not generate an empty report
+
+   **If using curl (no MCP server):**
+
+   a. Set up variables:
+   ```bash
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)